source/distributed/tria.cc

```diff
@@ -1842,6 +2361,27 @@ namespace parallel
           Assert(false, ExcInternalError());
         }
 
+      // load saved data, if any was stored
+      if (attached_count > 0)
+        {
+          data_transfer.load(parallel_forest, filename, attached_count);
+
+#  ifdef DEBUG
+          // The CellStatus of all stored cells should always be CELL_PERSIST.
+          data_transfer.unpack_cellstatus(local_quadrant_cell_relations);
```

Comment:
hm, why do you need this line only in debug mode?

Reply:
By loading a triangulation and collecting the <tt>quadrant_cell_relations</tt>, the <tt>CellStatus</tt> of each cell is <tt>CELL_PERSIST</tt>.

In this code snippet, we want to check that the previously saved triangulation fulfils the requirement. Since we won't check that in <tt>Release</tt>, we don't have to unpack the <tt>CellStatus</tt> beforehand.

Reply:
That's a prescription for a really difficult to find bug later on if you decide to use the cell status for something else. Your code will work in debug mode but not in release mode -- making it incredibly difficult to debug.

I would vote for doing the same thing in both debug and release mode, even though it will have a performance impact. We should really try hard for debug and release mode to produce no noticeable differences.

Reply:
Alright. I'll remove the <tt>#ifdef ... #endif</tt> routine to make sure the <tt>CellStatus</tt> is always unpacked. This should remove any ambiguity between both debug and release mode.